strawberry/subscriptions/protocols/graphql_transport_ws/handlers.py

```diff
@@ -117,6 +118,8 @@ async def handle_message(self, message: dict):
         await self.reap_completed_tasks()
 
     async def handle_connection_init(self, message: ConnectionInitMessage) -> None:
+        self.connection_params = message.payload or {}
```

Comment:
Do we actually want to default to an empty dict? The spec allows the connectionParams to be `null`/`None` and your typings in other places suggest that you wanted to allow `None` :eyes:

Reply:
I agree that it would be best to align closely to the established spec, so I will align all other instances of this to use `Optional[Dict[str, Any]]` and change empty dicts to `None`s :grin: